packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -40,6 +42,27 @@ export const isCameraOn = (state: CallAdapterState): boolean => {
   return false;
 };
 
+/* @conditional-compile-remove(video-background-effects) */
+/**
+ * @private
+ */
+export const startSelectedVideoEffect = async (adapter: CommonCallAdapter): Promise<void> => {
+  if (adapter.getState().selectedVideoBackgroundEffect) {
```

Comment:
is there a way to write a unit test for this? should we be providing warnings to the console that there was a problem starting a effect?